Bottom: 8b252cc73eb019e5c50fd6fa2b71195923f7589a
Top:    8a6441780449c507cf2c34e541327f641aa7b0f9
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 20:58:21 +0800

Refresh of vhost_net-split-out-datacopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 5e821de..4d2cec0 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -700,13 +700,10 @@ static void handle_tx(struct vhost_net *net)
 	vhost_disable_notify(&net->dev, vq);
 	vhost_net_disable_vq(net, vq);
 
-	if (vhost_sock_zcopy(sock)) {
-		printk("zcopy!\n");
+	if (vhost_sock_zcopy(sock))
 		handle_tx_zerocopy(net, sock);
-	} else {
-		printk("data copy!\n");
+	else
 		handle_tx_copy(net, sock);
-	}
 
 out:
 	mutex_unlock(&vq->mutex);
